### PR TITLE
Add text and style for Code of Conduct page - initial version

### PR DESCRIPTION
**Pull request by nverbic**
## Description
 
Add text and basic styles for Code of Conduct page
- Text copied from the Google Document: Website Contents

https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/23

## Change Type
- [ ] Bug Fix
- [x] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [ ] Other

## Screenshots

BEFORE:
image

AFTER:
image

image

image

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

**Comment by nverbic** [comment]
TODO: Add text for Code of Conduct for Mentors and Mentees -define where to add these.

File: code-of-conduct.html [hunk 1]
```diff
@@ -2,16 +2,47 @@
...
+               and foster professional relationships, regardless of age, gender, visible or invisible disability, ethnicity, gender expression, 
+               level of experience, education, socio-economic status, nationality, personal appearance, race, religion, sexual orientation or 
+               preferred programming language(s).</p>
         </div>
-
-
-        <div class="col-12 section global-code">
-            <h2 class="text-center">TBD</h2>
+        <div class="section">
+            <h3>Our Standards:</h3>
```
Comment (lilyndk) [inline code-of-conduct.html]: I guess these h3 should use h2 tags instead?

Comment (nverbic) [reply code-of-conduct.html]: I will change to h2  😎

Comment (nverbic) [reply code-of-conduct.html]: Screenshot with headings updated  to h2

image

**Review by lilyndk — APPROVED** [review]

**Pull request merged**